test/e2e/specs/wp-calypso-gutenberg-post-editor-spec.js

```diff
@@ -281,22 +281,43 @@ describe( `[${ host }] Calypso Gutenberg Editor: Posts (${ screenSize })`, funct
 			step( 'Can see the Earn blocks', async function () {
 				const gEditorComponent = await GutenbergEditorComponent.Expect( driver );
 				await gEditorComponent.openBlockInserterAndSearch( 'earn' );
-				assert.strictEqual(
-					await gEditorComponent.isBlockCategoryPresent( 'Earn' ),
-					true,
-					'Earn (Jetpack) blocks are not present'
-				);
+
+				// With deepEqual we're also testing the sort order they're shown in the
+				// inserter. If that's not desirable, we can use the sameMembers matcher.
+				assert.deepEqual( await gEditorComponent.getShownBlockInserterItems(), [
```

Comment:
Before it was just checking if the actual DOM element for the category tab was present. Category is not rendered as part of the inserter search UI anymore, though (see https://github.com/Automattic/wp-calypso/pull/47533/files#r525642380). 

We can either:

1. Check for the actual blocks. This is what I did for now;
2. Get rid of those tests?

Also, we I decided to use `deepEqual` in order to get sort order testing for free. If that's overkill, we can use `sameMembers`.

Thoughts?